modules/cloud-run/variables.tf

```diff
@@ -160,9 +160,7 @@ variable "revision_annotations" {
       max_scale = number
       min_scale = number
     }))
-    cloudsql_instances  = optional(list(string), [])
```

Comment:
I'm not sure we need to do this. For CR the right place for VPC connector is the annotations block. Maybe this part of the interface needs to stay different.

Reply:
For Cloud Run using V1 API yes, it lands in annotations, but if we check V2 API, it lands in dedicated fields. This is also the case for trigger configuration, which if we migrate CR to v2 API, we could have same API for CR and CF-v2

Reply:
But we decided to ignore v2 API since it's just a different API interface to the same thing, which is irrelevant when using the provider instead of naked API calls. Right? :)

I feel like we're chasing a red herring, like we did with GKE initially when using the same module for standard and autopilot.